index.js

```diff
@@ -22,6 +22,11 @@ function WaitForAll(count, allDone) {
 
 module.exports = function (PIXI)
 {
+    var PIXI_VERSION = Number(PIXI.VERSION.charAt(0));
```

Comment:
You could start using `const` and `let` instead of sticking with `var`

Reply:
This lib is not using babel, we probably can't without breaking some browser we currently support.